src/_pytest/python.py

```diff
@@ -561,7 +561,7 @@ def __init__(self, fspath, parent=None, config=None, session=None, nodeid=None):
     def _recurse(self, path):
         ihook = self.gethookproxy(path.dirpath())
         if ihook.pytest_ignore_collect(path=path, config=self.config):
-            return
+            return False
```

Comment:
This doesn't have anything to do with the fix, just thought I would keep the return value consistent.